.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,29 +3,31 @@
 
 ## Summary
 
+- *This work is behind a feature toggle (flipper): YES/NO*
 - *(Summarize the changes that have been made to the platform)*
 - *(If bug, how to reproduce)*
-- *(What is the solution, why is this the solution)*
-- *(Which team do you work for, does your team own the maintainence of this component?)*
-- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*
+- *(What is the solution, why is this the solution?)*
+- *(Which team do you work for, does your team own the maintenance of this component?)*
+- *(If introducing a flipper, what is the success criteria being targeted?)*
 
 ## Related issue(s)
-- department-of-veterans-affairs/va.gov-team#0000
-- *Link to ticket created in va.gov-team repo*
+
+- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*
 - *Link to previous change of the code/bug (if applicable)*
 - *Link to epic if not included in ticket*
 
-
 ## Testing done
 
+- [ ] *New code is covered by unit tests*
```

Comment:
Isn't this already covered by a lint check?

Reply:
@ericboehs we do have a code coverage check, but does it ever fail?

Reply:
Or are we ever notified if it drops? I was thinking this check would be more of an accountability check/reminder.

Reply:
It seems like I've seen it fail. It shouldn't be hard to test.

But I'm not going to block you on this, just thought I'd mention it